workers/unity/Packages/io.improbable.gdk.core/Utility/CommandLineUtility.cs

```diff
@@ -59,6 +59,11 @@ public bool TryGetCommandLineValue<T>(string key, ref T value)
             var desiredType = typeof(T);
             if (arguments.TryGetValue(key, out var strValue))
             {
+                if (string.IsNullOrEmpty(strValue))
```

Comment:
I think this will be hit when you do the following as well `+someArgument ""`.
I don't think there is a difference currently between "no" argument and an "empty" argument?

Reply:
@zeroZshadow Should I allow empty argument to pass or throw a different exception for empty argument?

Reply:
I think an empty string as a value is valid, so we'd need a different way to differentiate between a flag and an empty value. @jamiebrynes7 @paulbalaji @seanjparker Opinions?

Reply:
I can just set it to `null` for flags.

Reply:
`null` for flags and `string.Empty` for truly empty arguments seems reasonable to me